src/rng/astrolich/victrixLudorum/index.ts

```diff
@@ -0,0 +1,83 @@
+import { sampleSize, sample, random } from 'lodash/fp';
+import { victrixLudorum as attribution } from 'rng/attributions';
+import { formatAbilities, rollAbilities } from 'rng/shared/abilities';
+import { formatBody } from 'rng/shared/bodies';
+import { formatClass } from 'rng/shared/class';
+import { formatHabit } from 'rng/shared/habits';
+import { formatTrait } from 'rng/shared/traits';
+import { titledEntry, formatTitledEntry } from 'rng/shared/entries';
+import {
+  formatEquipment,
+  hasScroll,
+  rollArmor,
+  rollFoodAndWater,
+  rollScroll,
+  rollSilver,
+  rollStandardEquipment,
+} from '../../shared/equipment';
+import { rollHp, formatHp } from 'rng/shared/hp';
+import { formatName } from 'rng/shared/names';
+import { equipmentEntry } from 'rng/shared/entries';
+import { rollOmens, formatOmens } from 'rng/shared/omens';
+import tables from 'rng/tables';
+import { Character, TableEntry } from 'types/character';
+
+export const entry = (id: string): TableEntry => equipmentEntry(attribution, id)
+
+
+export const victrixLudorum = (): Character => {
+  const abilities = rollAbilities(1, -1, 1, 1);
+  const hp = rollHp(1, 8, abilities.toughness.score);
+  const omens = rollOmens(1, 2);
+
+  const generalEquipment = rollStandardEquipment();
+  const armor = rollArmor(3, hasScroll(generalEquipment));
+  const silver = rollSilver();
+  const foodAndWater = rollFoodAndWater();
+  const scrollCount = random(1,2);
```

Comment:
Karl rolled a Victrix with nine scrolls, so something is buggy :) Is a separate scroll roll actually part of the Victrix class? rollStandardEquipment() already includes possible scrolls from the regular equipment tables.